### DISCUSSION TITLE
Welcome to cperl-mode Discussions!

### DISCUSSION THREAD

**Discussion by HaraldJoerg** [HaraldJoerg/cperl-mode#10] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by smonff** [comment]
Hi, I am Sébastien from france.

I use your version of `cperl-mode` on Doom Emacs. It is very easy to integrate by adding the following to `~/.doom.d/packages.el`:

```elisp
(package! cperl-mode
  :recipe (:host github :repo "HaraldJoerg/cperl-mode"))
```

Then `doom sync` and later to check for update `doom upgrades`.

It is very useful for using with [Zydeco](https://zydeco.toby.ink/). Thanks for the work!

**Reply by HaraldJoerg** [reply]
Nice to hear!
There haven't been updates in a while because I'm currently working on old bugs in CPerl mode.  Most of these fixes haven't been merged into the master branch (some can't be easily applied due to the changes), so please open an issue if you find bad behavior!
